### PR TITLE
attempted to fix CI instructions. closes #10

### LINKED ISSUE #10: Build process
A few changes that need to be made to the build process:

1. all the HTML files must have a name like `2021-08-17-<whatever>.html` (note: please just hardcode the date, so that it doesn't change all the time)
2. the CI job should do this block https://github.com/plotly/plotlyjs.jl-docs/blob/master/.circleci/config.yml#L40 when the branch is `master` and should push the HTML build outputs to a new branch on this repo called `built`

### PR DESCRIPTION
**Pull request by sglyon**

### CODE REVIEW

**Comment by sglyon** [comment]
@nicolaskruchten does this resolve the issue from #10?

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
               git config user.email accounts@plot.ly
               git add *
               git commit -m "build of https://github.com/plotly/plotlyjs.jl-docs/commit/${CIRCLE_SHA1}"
-              git push --force git@github.com:plotly/plotlyjs.jl-docs.git master:built_ipynb
+              git push --force git@github.com:plotly/plotlyjs.jl-docs.git master:built
```
Comment (nicolaskruchten) [inline .circleci/config.yml:56]: so I think the `git push` a few lines above is what I need... I don't need the IPYNB files, but if you want to keep them in `built_ipynb` that's fine with me.

To fully resolve #10 I'll need the HTML file names to start with a date too though.

Comment (nicolaskruchten) [reply .circleci/config.yml:56]: Ah I see you did that already :)  https://github.com/plotly/plotlyjs.jl-docs/commit/89c2e494d9b384bb73dc512693bbf917cb3c0105

Comment (sglyon) [reply .circleci/config.yml:56]: yep -- already have the date :)

Also, while you were OOO I ran into some bugs in the jupyter pipeline. I have replaced it with a custom (<200 LoC) solution in pure julia. There are no more notebooks. 

Ha so I think that means I should delete this whole ipynb section of CI config and we shoudl be good to go?

Comment (nicolaskruchten) [reply .circleci/config.yml:56]: sure yeah delete the Ipynb section, and then I'm logging some more notes in #10 for small things that need adjustment before the output can be fed into the next part of the pipeline. We're super close :) Thanks!

**Comment by sglyon** [comment]
Hey @nicolaskruchten how does this look now?

I've taken your comments over in #10 and removed some of the unnecessary markup in the html output

**Comment by sglyon** [comment]
Merged to get the CI job to run and see if we can push to this repo

**Comment by nicolaskruchten** [comment]
OK, I've broken up the remaining items from #10 into #12 :)

**Pull request merged**